test/postman/dbp-test-collection.json

```diff
@@ -863,8 +963,7 @@
 					"raw": ""
 				},
 				"url": {
-					"raw": "http://{{url}}/actuator/routes",
```

Comment:
Why did we remove http here? will this work like this, in ci we are only giving the elb name not the protocol.

Reply:
The protocol was removed in order to enable testing with either http or https. As for CI, there is also an accompanying change to cater for this https://git.alfresco.com/platform-services/bamboo-build-dbp/merge_requests/16.

Reply:
Ok, thanks for the explanation.